extensions/amp-form/0.1/amp-form.js

```diff
@@ -216,6 +221,29 @@ export class AmpForm {
   }
 
   /**
+   * Returns form data as an object.
+   * @return {!Object}
+   * @private
+   */
+  getFormAsObject_() {
+    const data = {};
+    const inputs = this.form_.querySelectorAll('input,select,textarea');
```

Comment:
Haven't used `form.elements` before, but quick test it seems Safari has different implementation than Chrome, Safari included `fieldset`s but chrome didn't.

Is using this for now ok or should I do more research on `form.elements`?